src/metabase/driver/common.clj

```diff
@@ -394,3 +395,8 @@
     (if (nil? json-unfolding)
       true
       json-unfolding)))
+
+;; Driver-specific extra-info methods
+
+(defmethod driver/extra-info :postgres [_]
```

Comment:
It's strange that this exports all providers as postgres providers but those aren't marked as such. That list seems to invite additions but there's no mechanism for which ones are postgres, which ones mongo, etc. And I don't see how third party authors could extend it. That's an acceptable limitation for the time being i suppose, but seems strange that everything is essentially hardcoded to postgres.